src/core/public/overlays/flyout/flyout_service.tsx

```diff
@@ -91,6 +91,7 @@ export interface OverlayFlyoutStart {
 export interface OverlayFlyoutOpenOptions {
   className?: string;
   closeButtonAriaLabel?: string;
+  ownFocus?: boolean;
```

Comment:
TIL: the fact that this change did not requires a doc re-generation means that this type is not properly exported from core. I will take a look